src/EditorFeatures/Core/Implementation/Workspaces/EditorInfoBarService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Extensions;
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Workspaces
+{
+    internal class EditorInfoBarService : IInfoBarService
+    {
+        public void ShowInfoBarInActiveView(string message, params InfoBarUI[] items)
+        {
+            ShowInfoBarInGlobalView(message, items);
```

Comment:
Because I missed that ErrorErrorReportingService was exported as a factory from EditorErrorReportingServiceFactory when I was copying it.